skyportal/utils/gcn.py

```diff
@@ -340,6 +353,27 @@ def get_col(m, name):
     return skymap
 
 
+def get_occulted(url, nside=64):
+
+    m = Table.read(url, format='fits')
+    ra = m.meta.get('GEO_RA', None)
+    dec = m.meta.get('GEO_DEC', None)
+    error = m.meta.get('GEO_RAD', 67.5)
+
+    if (ra is None) or (dec is None) or (error is None):
+        return None
+
+    center = SkyCoord(ra * u.deg, dec * u.deg)
+    radius = error * u.deg
+
+    hpx = HEALPix(nside, 'ring', frame=ICRS())
```

Comment:
Are you sure "ring" is the right one? I'm guessing you'd want a filled-circle...